editor/code_editor.cpp

```diff
@@ -1852,6 +1859,12 @@ void CodeTextEditor::_apply_settings_change() {
 
 	text_editor->set_code_hint_draw_below(EDITOR_GET("text_editor/completion/put_callhint_tooltip_below_current_line"));
 
+#ifdef LINUXBSD_ENABLED
+	if (DisplayServer::get_singleton()->has_feature(DisplayServer::FEATURE_CLIPBOARD_PRIMARY)) {
+		text_editor->set_middle_mouse_paste_enabled(EDITOR_GET("text_editor/behavior/input/middle_mouse_button_paste"));
+	}
+#endif
```

Comment:
```suggestion
	text_editor->set_middle_mouse_paste_enabled(EDITOR_GET("text_editor/behavior/input/middle_mouse_button_paste"));
```

`FEATURE_CLIPBOARD_PRIMARY` is always set on X11 DisplayServer, so there's no need for this check.

Reply:
Done!